### DISCUSSION TITLE
The matrix format after using ToSparseTensor()

### DISCUSSION THREAD

**Discussion by ZhouAo-ZA** [pyg-team/pytorch_geometric#3653] [General]
When we want to use SpMM mode to calculate GCN, data needs to be converted to sparse matrix format using `ToSparcetensor`. I want to know whether the matrix is CSR, CSC or COO format before executing SpMM？Similarly, before executing `segment_csr` ?

**Comment by rusty1s** [comment]
`SparseTensor` can represent any sparse format (COO, CSR, CSC). For SpMM, it will convert to CSR.

**Reply by ZhouAo-ZA** [reply]
thanks, got it!
